test/client.spec.js

```diff
@@ -431,4 +440,53 @@ describe('Vue.js JHipster blueprint', () => {
             assert.fileContent('package.json', '"vuelidate"');
         });
     });
+    describe('Common module', () => {
```

Comment:
The test file is named `client.spec.js` but it doesn't really match anymore the tests being done here (that the whole blueprint, not just the client subgen).